include/odp/api/spec/packet_io_stats.h

```diff
@@ -86,6 +86,57 @@ typedef struct odp_pktio_stats_t {
 	uint64_t out_errors;
 } odp_pktio_stats_t;
 
+/**
+ * Packet IO statistics capabilities
+ */
+typedef struct odp_pktio_stats_capability_t {
```

Comment:
How about not duplicating this structure and reuse odp_pktio_stats_t

Reply:
The reasoning for adding a separate struct was to enable adding additional capabilities in the future, which may not be counters. E.g. are pause frames included in the counters.

Reply:
Benefits of the bit field are saved memory space (8 vs 80bytes) and easy comparison of multiple flags (by using all_counters).

The stats capa structure allows to combine capabilities for all pktio statistics, and also potential relation between those (e.g. "both interface and queue level stats supported, but not simultaneously").